docs/05-PubsubForValidators.md

```diff
@@ -0,0 +1,58 @@
+# The x/pubsub Module for Validators
+
+Before Sommelier v7, the Steward Registry was used to record the CA and server endpoint of all steward processes. This database was then used by Strategy Providers (SPs) to send requests to each steward instance. 
+
+The Steward Registry has been deprecated in favor of a (mostly) generalized pubslisher/subscriber registry Cosmos SDK module, [x/pubsub](https://github.com/peggyjv/sommelier/tree/main/x/pubsub).
+
+For more information on x/pubsub itself, please refer to the [protocol documentation]()
+
+## Registering Steward as a Subscriber
+
+A `Subscriber` is an entity representing the identity of your steward instance. It contains your Sommelier address (either the validator or delegate address), server Certificate Authority, and the Steward server endpoint to which Strategy Providers (publishers) will send requests. 
+ 
+### Prerequisites
+
+- The latest `sommelier` binary release
+- Key, CA, and self-signed certificate generated using the steps in [the previous doc](./04-GeneratingCertificates.md).
+- Your validator or delegate Sommelier address registered on the gravity module
+- A domain and port that will or already points to your steward serverwith
+- Firewall configured to allow requests to the former
+
+### Steps
+
+A subscriber can be created using the following CLI command
+
+```bash
+sommelier tx pubsub add-subscriber <subscriber_address> [<ca_cert_file>] [<push_url>]
```

Comment:
Might want to point out that the `push_url` is the URL for your public-facing steward instance. Maybe refer to `subscriber_address` specifically as orchestrator address as well. Also I know we mention they're not really optional at the bottom, but maybe we just leave the brackets off the example here.